lib/resources/storage/mobile_storage.dart

```diff
@@ -11,4 +14,31 @@ class StorageProvider {
       return null;
     }
   }
+
+  static Future<bool> uploadImage(
+      BuildContext context, Uint8List file, String ref) async {
+    try {
+      final StorageReference reference =
+          FirebaseStorage.instance.ref().child(ref);
+
+      final StorageUploadTask uploadTask = reference.putData(file);
+      await uploadTask.onComplete;
+      if (uploadTask.isSuccessful) {
+        return true;
+      } else {
+        return false;
```

Comment:
Print the error to the console when this fails (we'll change the logging soon, but for now that's good enough), and below in the catch block too. Might be easier and cleaner to just use the onComplete callback (`await reference.putData(file).onComplete(...)`)

Reply:
There is no `onComplete()` method but i found a work around